script/gcp-cleanup

```diff
@@ -9,6 +9,7 @@ export GOOGLE_APPLICATION_CREDENTIALS="/openebs/e2e-gcp/key.json"
 echo "cleanup"
 mkdir ~/logs
 cp .kube/clusters ~/logs
+cp /kube/vpc ~/.logs
```

Comment:
This line should be like `cp .kube/vpc ~/logs`

Reply:
Thanks! @chandankumar4